core/client/Cargo.toml

```diff
@@ -32,6 +32,7 @@ header-metadata = { package = "substrate-header-metadata", path = "header-metada
 [dev-dependencies]
 env_logger = "0.7.0"
 tempfile = "3.1.0"
+client-db = { package = "substrate-client-db", path = "./db", features = ["kvdb-rocksdb"] }
```

Comment:
I added this in order to force the `kvdb-rocksdb` feature, otherwise the test fails.
Alternatively, I could maybe change the test to use an in-memory database.